programs/tensorflowDetection2D/main.cpp

```diff
@@ -0,0 +1,160 @@
+/*
+ * ************************************************************
+ *      Program: Tensorflow Detection 2D Module
+ *      Type: main.cpp
+ *      Author: David Velasco Garcia @davidvelascogarcia
+ * ************************************************************
+ */
+
+// Librerias
+
+#include <iostream>
+#include <yarp/os/all.h>
+#include <yarp/sig/all.h>
+#include <opencv2/opencv.hpp>
+#include <yarp/sig/Image.h>
+#include <yarp/os/Time.h>
+#include <tensorflow/core/platform/env.h>
+#include <tensorflow/core/public/session.h>
+#include "TensorflowDetection2D.hpp"
+#include "TensorflowSessionTest.h"
+
+// Espacios de nombres
+
+using namespace yarp::os;
+using namespace yarp::sig;
+using namespace yarp::sig::draw;
+using namespace cv;
+using namespace std;
+using namespace tensorflow;
+
+// Variables
+
+int yarpserver_ok=0;
+string source_video="/home/tiagoentrenamiento/Vídeos/tiago.mp4";// Test: /home/tiagoentrenamiento/Vídeos/tiago.mp4
+string labels = "models/ssd_mobilenet_v1_egohands/labels_map.pbtxt";
+string graph = "models/ssd_mobilenet_v1_egohands/frozen_inference_graph.pb";
+
+int main(){
+
+  // Mensaje de bienvenida
+  system("clear");
```

Comment:
Avoid transitive headers (https://github.com/roboticslab-uc3m/developer-manual/issues/19): `std::system` is declared in `<cstdlib>`, hence an `#include` directive is missing. You can find this out in [cppreference](https://en.cppreference.com/mwiki/index.php?title=Special%3ASearch&search=system).

Reply:
Something like that 27143bca466e76f32fc5a403327422467027a301?

Reply:
Sure, thanks.